src/bin/travis/testCentaurLocal.sh

```diff
@@ -36,4 +36,8 @@ git clone https://github.com/broadinstitute/centaur.git
 cd centaur
 git checkout ${CENTAUR_BRANCH}
 cd ..
+# All tests use ubuntu:latest - make sure it's there before starting the tests 
+# because pulling the image during some of the tests would cause them to fail 
```

Comment:
Do you think we could build a retry for docker failures in Centaur itself?  Or better to have something like that in Cromwell?